ssh/tailssh/incubator.go

```diff
@@ -252,6 +274,7 @@ func (ss *sshSession) launchProcess() error {
 
 	ci := ss.conn.info
 	cmd.Env = append(cmd.Env,
```

Comment:
Style thing: rather than append to Env, could you make this an assignment? `cmd.Env = []string{...}`. That way it's 100% obvious what the child's environment will be, without having to trace down the chain of newIncubatorCommand and exec.CommandContext to figure out if they inherit the current env.

(this is important because sanitizing the environment has some security implications, so being able to quickly reason about what env is getting set matters a lot)